src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -1,5 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>InstallmentOppStageName__c</fullName>
+        <description>The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation.</inlineHelpText>
+        <label>Stage Name for Installment Opportunities</label>
+        <length>50</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>IsEnhancementEnabled__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>This option enables Enhanced Recurring Donations, and offers you the latest Recurring Donation features. &lt;b&gt;WARNING&lt;/b&gt;: When you enable this option, NPSP changes your existing Recurring Donation records and the Recurring Donations object to match the data model for Enhanced Recurring Donations. &lt;b&gt;This is an irreversible process&lt;/b&gt;. For complete details, see the NPSP product documentation.</description>
```

Comment:
@meena-sfdc HTML tags are not supported in HelpText or Description, so these can be removed.

Reply:
Thanks, @force2b

Reply:
@force2b , do we plan to display field's Help Text on any custom page?

Reply:
NPSP Settings page do sometimes display the InlineHelpText value, but we also use custom labels. In the case of this particular field though, there is no plan to render it directly in the NPSP Settings UI. There will be an enablement page, but it will not be a checkbox.